### DISCUSSION TITLE
Projects using Eta

### DISCUSSION THREAD

**Discussion by bgub** [eta-dev/eta#85] [General]
There are quite a few projects using Eta, and it would be great to feature some of them.

Notably, [Docusaurus 2](https://github.com/facebook/docusaurus) uses Eta for SSR generation. I'll see whether I can list them.

**Comment by willemmulder** [comment]
@nebrelbug I've been using eta for these production sites: https://jouwplekaandemuur.nl/ and https://jouwfotoopkarton.nl/. I'm not sure if you want commercial sites in your readme, but if so I'm happy to add them! :-)

**Comment by shadowtime2000** [comment]
@willemmulder @nebrelbug is currently on a 2 year service opportunity in which he will not be able to help maintain Eta. I think we are looking for open source projects so people can look at how those projects use Eta.

**Comment by willemmulder** [comment]
@shadowtime2000 Makes sense. Thanks for the reply!
